addons/point_of_sale/static/src/app/models/data_service.js

```diff
@@ -62,29 +61,32 @@ export class PosData extends Reactive {
         });
     }
 
-    async resetIndexedDB() {
-        await this.indexedDB.reset();
-    }
-
     get databaseName() {
         return `config-id_${odoo.pos_config_id}_${odoo.access_token}`;
     }
 
-    initIndexedDB() {
-        // In web tests info is not defined
-        const models = Object.entries(this.opts.databaseTable).map(([name, data]) => {
-            return [data.key, name];
+    initIndexedDB(relations) {
+        // This method initializes indexedDB with all models loaded into the PoS. The default key is ID.
+        // But some models have another key configured in data_service_options.js. These models are
+        // generally those that can be created in the frontend.
+        const models = Object.keys(relations).map((model) => {
+            const key = this.opts.databaseTable[model]?.key || "id";
+            return [key, model];
         });
-        this.indexedDB = new IndexedDB(this.databaseName, INDEXED_DB_VERSION, models);
-    }
 
-    deleteDataIndexedDB(model, uuid) {
-        this.indexedDB.delete(model, [{ uuid }]);
+        return new Promise((resolve) => {
+            this.indexedDB = new IndexedDB(this.databaseName, INDEXED_DB_VERSION, models, resolve);
```

Comment:
I can't find docs about the constructor parameters of `IndexedDB`, can you point me to a link?